gpu_benchmark/benchmark.sh

```diff
@@ -0,0 +1,86 @@
+#!/bin/bash
+
```

Comment:
Could we add a comment just to say which repo the git commit referenced is from?